nomad/pool.go

```diff
@@ -175,6 +175,19 @@ func (p *ConnPool) Shutdown() error {
 	return nil
 }
 
+// ReloadTLS reloads TLS configuration on the fly
+func (p *ConnPool) ReloadTLS(tlsWrap tlsutil.RegionWrapper) {
+	p.Lock()
+	defer p.Unlock()
+
+	oldPool := p.pool
+	for _, conn := range oldPool {
+		conn.Close()
```

Comment:
this is ignoring the error from conn.Close(), but so is the rest of this file. Need some context from @dadgar or @schmichael as to why close errors are ignored.

Reply:
There's really nothing useful to do with errors returned from `conn.Close()`. The connection will always be closed, and the error is never worth terminating the entire process over. I think even logging these errors would only lead to confusion.

Reply:
As far as I know there's nothing useful to do with Close errors. In some circumstances logging them might be useful, but I can't imagine what actionable information errors from this Close could contain? I don't think it's even useful for catching programming errors like closing closed connections as there's always the possibility the connection may close concurrently due to peer disconnects or other issues.

If somebody has an example of a useful Close error I'd love to know it!